modules/gatsby-theme-ocular/src/gatsby-node/on-create-node/process-nodes-json.js

```diff
@@ -15,7 +16,11 @@ function processEntry(chapter, entry, docNodes) {
     )();
     return;
   }
-  const relPath = entry.entry.replace(/^\//, '').replace(/\.[^/.]+$/, '').replace(/\/$/, '').replace('/README', '');
+  let relPath = entry.entry.replace(/^\//, '').replace(/\.[^/.]+$/, '').replace(/\/$/, '').replace('/README', '');
```

Comment:
Since you changed it, can you add a comment in front of this inscrutable line please. What does it do?

Reply:
Haven't actually changed it, just switched from `const` to `let`. The last part regarding `README` is understandable, but I am unsure why do we have these replacements before. In what cases we use this?